src/Modules/Common.lua

```diff
@@ -656,7 +656,22 @@ function formatNumSep(str)
 			return m
 		end
 		local x, y, minus, integer, fraction = str:find("(-?)(%d+)(%.?%d*)")
-		if main.showThousandsSeparators then
+		if main.lowerPrecisionDisplay and #integer > 3 then
+			if fraction == "" then
+				fraction = "."..integer:sub(-1)
+			else
+				fraction = "."..integer:sub(-#(fraction or "  ") + 1)
+			end
+			if #integer > 12 then
+				return colour..minus..integer:sub(1, #integer - 12)..fraction:gsub("%.", main.decimalSeparator).." T"
+			elseif #integer > 9 then
+				return colour..minus..integer:sub(1, #integer - 9)..fraction:gsub("%.", main.decimalSeparator).." B"
```

Comment:
Can we switch that to `G` like [SI prefix](https://en.wikipedia.org/wiki/International_System_of_Units#Prefixes) and not use billion which have different meaning if you use short or long scale [Wiki](https://en.wikipedia.org/wiki/Long_and_short_scales)